### PR TITLE
Improve how code insights helps

### PR DESCRIPTION
**Pull request by rrhyne**

### CODE REVIEW

File: content/strategy-goals/strategy/use_cases/fixing_security_vulnerabilities.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The following are real, anonymized quotes from customers today on how we’re al
 
 [Engineering Leader](https://docs.google.com/presentation/d/1aQhcWoWd_LJXdAgEn7JBGnZV5pfN6UJyct2VV-ZiTXI/edit#slide=id.ge9b93ff711_0_46)
 
-- Access Code Insights and set up Code Monitoring alerts to be notified when risky patterns, secrets, or other known vulnerabilities are introduced into the codebase
+- Create Code Insights to track remediation and set up Code Monitoring alerts to be notified when risky patterns, secrets, or other known vulnerabilities are introduced into the codebase
```
Comment (rrhyne) [inline content/strategy-goals/strategy/use_cases/fixing_security_vulnerabilities.md]:
```suggestion
- Create Code Insights to track remediation of the issue across affected repositories and set up Code Monitoring alerts to be notified when risky patterns, secrets, or other known vulnerabilities are introduced into any codebase
```

**Comment by jasonyavorsky** [comment]
Probably better for @anna-mmikhova to review but looks good to me.

**Review by jasonyavorsky — APPROVED** [review]

File: content/strategy-goals/strategy/use_cases/fixing_security_vulnerabilities.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The following are real, anonymized quotes from customers today on how we’re al
 
 [Engineering Leader](https://docs.google.com/presentation/d/1aQhcWoWd_LJXdAgEn7JBGnZV5pfN6UJyct2VV-ZiTXI/edit#slide=id.ge9b93ff711_0_46)
 
-- Access Code Insights and set up Code Monitoring alerts to be notified when risky patterns, secrets, or other known vulnerabilities are introduced into the codebase
+- Create Code Insights to track remediation of the issue across affected repositories and set up Code Monitoring alerts to be notified when risky patterns, secrets, or other known vulnerabilities are introduced into any codebase
```
Comment (rrhyne) [inline content/strategy-goals/strategy/use_cases/fixing_security_vulnerabilities.md:47]:
```suggestion
- Create Code Insights to track remediation of issues across affected repositories and set up Code Monitoring alerts to be notified when risky patterns, secrets, or other known vulnerabilities are introduced into any codebase
```

**Review by Joelkw — APPROVED** [review]
Thanks! Haven't been able to keep up with the new handbook pages yet, appreciate this!

**Pull request closed without merging**